conifer/converters/ydf.py

```diff
@@ -12,34 +13,48 @@
 def convert(model: ydf.GenericModel) -> ConiferModel:
     """Converts a YDF model into a Conifer model."""
 
-    if model.task() != ydf.Task.CLASSIFICATION:
+    if not model.task() in [ydf.Task.CLASSIFICATION, ydf.Task.ANOMALY_DETECTION]:
         raise ValueError(
-            f"Classification YDF model expected. Instead, found task {model.task()!r}"
+            f"Classification or Anomaly Detection YDF model expected. Instead, found task {model.task()!r}"
         )
-    if isinstance(model, ydf.GradientBoostedTreesModel):
+    if isinstance(model, (ydf.GradientBoostedTreesModel, ydf.IsolationForestModel)):
         return _convert_gbt(model)
     raise ValueError(f"Not supported YDF model type {type(model)}")
 
 
-def _convert_gbt(model: ydf.GradientBoostedTreesModel) -> ConiferModel:
+def _convert_gbt(model: Union[ydf.GradientBoostedTreesModel, ydf.IsolationForestModel]) -> ConiferModel:
     """Converts a YDF GBT model into a Conifer model."""
 
+    gbtm = isinstance(model, ydf.GradientBoostedTreesModel)
+    ifm = isinstance(model, ydf.IsolationForestModel)
     src_trees = model.get_all_trees()
     column_idx_to_feature_idx = _feature_mapping(model)
     max_depth = _get_max_depth(src_trees)
-    initial_predictions = model.initial_predictions().tolist()
-    label_classes = model.label_classes()
+    label_classes = model.label_classes() if gbtm else [0]
+    initial_predictions = model.initial_predictions().tolist() if gbtm else [0] * len(label_classes)
     num_trees_per_iter = len(initial_predictions)
     num_iterations = model.num_trees() // num_trees_per_iter
 
```

Comment:
Great hack :).

I am going to add the `model.num_examples_per_trees` method for the next release of YDF.
In the meantime, you can access the number of example per trees as follow:

```python
num_examples_per_trees = model.get_tree(tree_idx=0).root.value.num_examples_without_weight
```

Reply:
Will be available in the next YDF release: https://github.com/google/yggdrasil-decision-forests/commit/b1807dfb13cf55302cbc3146ca59574d40c0acff